stytch/b2b/scim/types.go

```diff
@@ -79,6 +84,7 @@ type SCIMAttributes struct {
 	Emails              []Email              `json:"emails,omitempty"`
 	PhoneNumbers        []PhoneNumber        `json:"phone_numbers,omitempty"`
 	Addresses           []Address            `json:"addresses,omitempty"`
+	Ims                 []IMs                `json:"ims,omitempty"`
```

Comment:
What is Ims? This seems like a really obscure name. We should add an override for it.

Reply:
@ollie-stytch @bgier-stytch any thoughts on the naming here?

Reply:
It's an instant messaging address for the user. `InstantMessagingAddress` but ims is what it's referred to as in the SCIM spec https://datatracker.ietf.org/doc/html/rfc7643#section-4.1.2

Reply:
Okay, if it's based on a spec, that's all good! No concern, then. Just wanted to check since it wasn't familiar to me.